ovos_padatious/opm.py

```
@@ -92,8 +93,9 @@ def __init__(self, bus, config):
         self.bus = bus
 
         core_config = Configuration()
-        self.lang = core_config.get("lang", "en-us")
+        self.lang = standardize_lang_tag(core_config.get("lang", "en-US"))
         langs = core_config.get('secondary_langs') or []
+        langs = [standardize_lang_tag(l) for l in langs]
```

Ruff

Ambiguous variable name: `l`

(E741)